upgrade/security_key_generator/MaterialGenerator.cpp

```diff
@@ -1,9 +1,9 @@
 #include "upgrade/security_key_generator/MaterialGenerator.hpp"
-#include "crypto/micro-ecc/uECC.h"
 #include "generated/echo/UpgradeKeys.pb.hpp"
 #include "hal/generic/FileSystemGeneric.hpp"
 #include "infra/stream/StdVectorOutputStream.hpp"
 #include "infra/syntax/ProtoFormatter.hpp"
+#include "uECC.h"
```

Comment:
It would be nicer if the path could be more explicit.

Reply:
Since we are used to that, true. But for mbedtls, args and all other FetchContent'd dependencies we have also stripped the path to what the package itself provides. So, I prefer to keep it this way. It would also make switching to another package source (vcpkg, Conan) easier, since this is the colloquial path.